### PR TITLE
Feature/user groups

### PR DESCRIPTION
**Pull request by claireriley**
For the release of user groups.

### CODE REVIEW

**Review by sowmyaraj92 — COMMENTED** [review]
@claireriley This looks great! 👏 🏆 I have added a few comments. Once we update the screenshots, please do check/update the existing ones on the document to match the latest design changes.

File: src/pages/docs/administration/managing-your-team/user-groups.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+#### Managing roles on workspaces and Postman entities
+
+You can control a user group's access to individual workspaces, collections, APIs, environments, mock servers, and monitors. For more information on managing workspaces, see [Sharing workspaces](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#sharing-workspaces).
+
+For collections, APIs, environments, mock servers, and monitors, navigate to the entity in Postman and in the left sidebar, select **...** > **Manage roles**.
+
+SCREENSHOT MANAGE ROLES ON COLLECTION WITH USER GROUP SELECTED
+
+Use the search bar to add the user group, then give the group **Editor** or **Viewer** permissions with the dropdown. You can also remove a user group by selecting the **X** to the right of it in the list. Click **Save** to confirm changes.
```
Comment (sowmyaraj92) [inline src/pages/docs/administration/managing-your-team/user-groups.md:94]: I see **Save Changes** and **Save** used, do cross-check once you update the screenshots.

File: src/pages/docs/administration/managing-your-team/user-groups.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+<img alt="Create user group button" src="https://assets.postman.com/postman-docs/manage-team-create-group-button.jpg"/>
+
+Give your user group a name and description, add your team members, and select the team roles you would like to assign to the group, then **Create Group**.
+
+<img alt="Create user group form" src="https://assets.postman.com/postman-docs/manage-team-create-group-form.jpg" width="500px"/>
+
+> Team members will receive an email and in-app notification when added to a user group.
+
+<!-- -->
+> If a team member with a support-only account (admin and/or billing) is given the **Developer** role via a user group they are in, they will automatically take up an available paid spot on your Postman team. If no spots are available, you will not be able to grant access to the group with that team member.
```
[the remark this answers is not in the document]
Comment (claireriley) [reply src/pages/docs/administration/managing-your-team/user-groups.md:56]: Yep I'm working on the docs for this feature too, I'll look into this for its release later this week.

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ In Postman you can assign roles and permissions to provide access control.
 
 ## Roles in Postman
 
-As a team admin, you have the power to define Postman access at the team, workspace, collection, and API level. You can utilize Postman's role-based access control system to limit visibility of team resources, define your development workflow, and provide access to administrative and billing personnel.
+As a team admin, you have the power to define Postman access at the team, workspace, collection, and API level. You can utilize Postman's role-based access control system to limit visibility of team resources, define your development workflow, and provide access to administrative and billing personnel. You can manage access by individual, or, if you are on a [Postman Enterprise plan](https://www.postman.com/pricing), with [user groups](/docs/administration/managing-your-team/user-groups/).
```
[the remark this answers is not in the document]
Comment (claireriley) [reply src/pages/docs/collaborating-in-postman/roles-and-permissions.md:48]: Rephrased it, let me know and we can take a broader look at the page later on

**Review by ArjunSingh-PM — COMMENTED** [review]
@claireriley The changes look good. 

As discussed, let us use the term 'groups' instead of 'user groups'

**Review by sowmyaraj92 — COMMENTED** [review]
@claireriley This looks good! 👏

File: src/pages/docs/collaborating-in-postman/using-workspaces/managing-workspaces.md [hunk 4]
```diff
@@ -55,15 +55,15 @@ You can access public workspaces via the [**Workspaces** category in the API Net
 
 ## Sharing workspaces
 
-You can share workspaces with collaborators by inviting them. If you invite a collaborator to a personal workspace, Postman will convert it to a team workspace.
+You can share workspaces with collaborators by inviting them individually or via a [user group](/docs/administration/managing-your-team/user-groups/). If you invite a collaborator to a personal workspace, Postman will convert it to a team workspace.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/collaborating-in-postman/using-workspaces/managing-workspaces.md:58]: Let us refer to this as a group rather than a user group

**Review by ArjunSingh-PM — COMMENTED** [review]
Added a few comments.

File: src/pages/docs/administration/managing-your-team/user-groups.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+* [Next steps](#next-steps)
+
+## Creating a group
+
+As a team admin, you can create a group by navigating to Postman, selecting **Home** in the upper-left corner, then **Manage Team** on the left side. Alternatively, select **Team** in the upper-right corner > **Manage Team**.
+
+Above your list of team members, select **Groups** > **Create Group**.
+
+<img alt="Create group button" src="https://assets.postman.com/postman-docs/overall-group-view.jpg"/>
```
[the remark this answers is not in the document]
Comment (claireriley) [reply src/pages/docs/administration/managing-your-team/user-groups.md:49]: okay it kept changing while I was working on this

File: src/pages/docs/collaborating-in-postman/using-workspaces/managing-workspaces.md (hunk 4 above)
Comment (claireriley) [reply src/pages/docs/collaborating-in-postman/using-workspaces/managing-workspaces.md]: This was updated prior to this comment, let me know if it's not showing for you

**Review by ArjunSingh-PM — APPROVED** [review]
Thanks, @claireriley! The changes look good!

I have shared a couple of screenshots for managing roles on groups for a workspace and inviting groups to a workspace

**Review by sowmyaraj92 — APPROVED** [review]
This looks good! You have pulled it off nicely @claireriley 👏

**Pull request merged**